### DISCUSSION TITLE
"ALAN Conversion Guide", AsciiDoc Version

### DISCUSSION THREAD

**Discussion by tajmone** [alan-if/alan-docs#104] [General]
@thoni56,

I've ported the ALAN _Conversion Guide_ to AsciiDoc — sorry but I had to do some force pushing during the last updates, to clean up the history here and there, but the alpha man branch was rebased without conflicts.

I've taken liberty to fix some typos and polish a few sentences as I was porting and applying styles according to our guidelines, I hope you don't mind.
I've also left some `@TODO`, `@FIXME`, `@NOTE`, etc. comments with possible pending tasks and problematic text.

I'd now like to move the original ODT and PDF files of the _Conversion Guide_ in the [`_assets-src/original-docs/`][_assets-src/original-docs/] with all the other original upstream docs that were ported to ADoc.

I have a question about the _Conversion Guide_ folder and file names, which currently are:

- `conversion/`
- `conversion/conversion.asciidoc`
- `conversion/conversion.html`

I think that having a `conversion/` folder can be misleading and confusing, newbies might thing it's part of the conversion toolchain. The file names of the documents are not quite revealing about the nature of the document either.

What about changing `conversion` to something like:

- `conv-guide`
- `conversion-guide`
- `alan2conversion`
- `alan2conv`

etc., i.e. something which is more indicative of what the document is all about? Any proposals are welcome, but let's not keep `conversion`.

[_assets-src/original-docs/]: https://github.com/alan-if/alan-docs/tree/master/_assets-src/original-docs

**Comment by thoni56** [comment]
Thanks. Force-pushing is not a big problem, most of the time. I see myself doing that fairly often now that I work more intensely on these projects, primarily alanif/alan, of course.

Moving original sources to assets is a good move.

I agree on the folder name, and if you want my vote it would go to `alan2conversion`. I suppose `alan2to3conversion` would be another, very future-proof, name, but I guess it's unnecessary at this point. We can do that rename when Alan v4 is on the horizon ;-)
